src/Resources/ResourceManager/Utilities/BicepUtility.cs

```diff
@@ -131,7 +131,7 @@ public static void PublishFile(string bicepFilePath, string target, string docum
                 throw new AzPSArgumentException(Properties.Resources.InvalidBicepFilePath, "File");
             }
 
-            string bicepPublishCommand = $"publish {GetQuotedFilePath(bicepFilePath)} --target '{target}'";
+            string bicepPublishCommand = $"publish {GetQuotedFilePath(bicepFilePath)} --target {GetQuotedFilePath(target)}";
```

Comment:
Should we do the same for `documentationUri` below?

Reply:
Well actually that one is a URI so we should be fine.

Reply:
Actually, that one is a URI so it might just work.

Reply:
Nope, it gave an even weirder error:
```
Publish-AzBicepModule: Cannot bind parameter 'DocumentationUri' to the target. Exception setting "DocumentationUri": "Cannot perform operation because operation "GetPropertyOrFieldExpr" is not valid. Remove operation "GetPropertyOrFieldExpr", or investigate why it is not valid."
```

Reply:
a8d75d4617